ethportal-api/src/types/cli.rs

```diff
@@ -346,7 +346,7 @@ pub fn network_parser(network_string: &str) -> Result<Arc<NetworkSpec>, String>
 pub fn subnetwork_parser(subnetwork_string: &str) -> Result<Arc<Vec<Subnetwork>>, String> {
     let subnetworks = subnetwork_string
         .split(',')
-        .map(Subnetwork::from_str)
+        .map(Subnetwork::from_cli_arg)
```

Comment:
Maybe add in `pub fn subnetwork_parser(subnetwork_string: &str) -> Result<Arc<Vec<Subnetwork>>, String> {` a check if all the submitted `subnetwork`'s are active/supported or not. And through an error if the network isn't support yet

Reply:
+1 to this